library.properties

```diff
@@ -1,5 +1,5 @@
 name=EmotiBit ArduinoFilters
-version=1.0.0
+version=1.0.1
```

Comment:
Since `M_PI` has been changed to `PI`, doesn't that mean any other code depending on this repo is now going to break because they don't know what `PI` is? 
Or, maybe, every other Arduino code uses the `PI` in Arduino and cppApp code uses `PI` in mock?